### PR TITLE
Add documentation for todo.get_items service

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Add documentation for todo.list_items service.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103285
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/todo.markdown [hunk 1]
```diff
@@ -34,6 +34,27 @@ incomplete items in the list.
 Some To-do List integrations allow Home Assistant to manage the To-do Items in the list. The
 services provided by some To-do List entities are described below or you can read more about [Service Calls](/docs/scripts/service-calls/).
 
+
+### Service `todo.list_items`
+
+Get To-do items from a To-do list. A To-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector) and the `data` payload supports the following fields:
```
Comment (c0ffeeca7) [inline source/_integrations/todo.markdown]:
```suggestion
Get to-do items from a to-do list. A to-do list `target` is selected with a [target selector](/docs/blueprint/selectors/#target-selector). The `data` payload supports the following fields:
```

File: source/_integrations/todo.markdown [hunk 2]
```diff
@@ -34,6 +34,27 @@ incomplete items in the list.
...
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | --------|
+| `status` | yes | Only return To-do items with this status. |  `needs_action`, `completed`
```
Comment (c0ffeeca7) [inline source/_integrations/todo.markdown]:
```suggestion
| `status` | yes | Only return to-do items with this status. |  `needs_action`, `completed`
```

File: source/_integrations/todo.markdown [hunk 3]
```diff
@@ -34,6 +34,27 @@ incomplete items in the list.
...
+
+This is a full example that returns all To-do items with any status:
```
Comment (c0ffeeca7) [inline source/_integrations/todo.markdown]:
```suggestion
This is a full example that returns all to-do items with any status:
```
I realize to-do is capitalized elsewhere in this text. But I think shouldn't be. I will fix this later in a separate PR.

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM, @allenporter :+1:

**Comment by allenporter** [comment]
Updated based on discussion in core PR to rename the service to `todo.get_items`

**Review by c0ffeeca7 — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @allenporter 👍 

../Frenck

**Pull request merged**